apps/common/feathers/client.js

```diff
@@ -0,0 +1,15 @@
+const feathers = require('@feathersjs/feathers');
+const rest = require('@feathersjs/rest-client');
+
+const app = feathers();
+
+// Connect to the same as the browser URL (only in the browser)
+const restClient = rest();
+
+// eslint-disable-next-line no-undef
+if (window && window.fetch) {
```

Comment:
This'd be a good place to use a [fetch polyfill](https://github.com/github/fetch). It'd also make the import nicer.

Reply:
yar -- i ignored here because this is only client-side code